packages/gatsby/src/query/queue.js

```diff
@@ -99,7 +99,15 @@ const processBatch = async (queue, jobs, activity) => {
   if (numJobs === 0) {
     return Promise.resolve()
   }
-  const runningJobs = jobs.map(job => pushJob(queue, job, activity))
+
+  const runningJobs = jobs.map(job =>
+    pushJob(queue, job).then(v => {
```

Comment:
this merge removed here the `activity` param which is needed in `pushJob`:

https://github.com/gatsbyjs/gatsby/blob/88e7c7f1a889e2c756e572878385e54165770c94/packages/gatsby/src/query/queue.js#L84-L90